test/rekt/features/channel/features.go

```diff
@@ -93,3 +95,54 @@ func DeadLetterSink() *feature.Feature {
 
 	return f
 }
+
+func EventTransformation() *feature.Feature {
+	f := feature.NewFeature()
+	lib := feature.MakeRandomK8sName("lib")
+	channel1 := feature.MakeRandomK8sName("channel 1")
+	channel2 := feature.MakeRandomK8sName("channel 2")
+	prober := eventshub.NewProber()
+	prober.SetTargetResource(channel_impl.GVR(), channel1)
+
+	f.Setup("install events", eventlibrary.Install(lib))
+	f.Setup("use events cache", prober.SenderEventsFromSVC(lib, "events/three.ce"))
+	f.Setup("register event expectation", func(ctx context.Context, t feature.T) {
+		if err := prober.ExpectYAMLEvents(eventlibrary.PathFor("events/three.ce")); err != nil {
```

Comment:
you don't need to do this step inside a setup function, it can happen in the the main function

Reply:
What would I do with the error though? I don't have a T to assert with